var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -298,4 +297,16 @@ def cmake_args(self):
                 cmake_args.append(
                     '-DVTK_MODULE_ENABLE_VTK_IOMotionFX:BOOL=OFF')
 
+        # -no-ipo prevents an internal compiler error from multi-file
+        # optimization (https://github.com/spack/spack/issues/20471)
+        if '%intel' in spec:
+            compile_flags.append('-no-ipo')
+
+        if compile_flags:
+            compile_flags = ' '.join(compile_flags)
+            cmake_args.extend([
+                '-DCMAKE_C_FLAGS="{}"'.format(compile_flags),
+                '-DCMAKE_CXX_FLAGS="{}"'.format(compile_flags)
```

Comment:
```suggestion
                '-DCMAKE_C_FLAGS={0}'.format(compile_flags),
                '-DCMAKE_CXX_FLAGS={0}'.format(compile_flags)
```
Needed for Python 2.6 support